src/dfa.rs

```diff
@@ -1644,8 +1688,7 @@ impl Byte {
 impl fmt::Debug for State {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.debug_struct("State")
-         .field("flags", &self.flags)
-         .field("insts", &self.insts)
+         .field("data", &self.data)
```

Comment:
Could you make this impl show something a bit more useful? It would be nice if it kept roughly the same information content as it was before.